packages/ui/package.json

```diff
@@ -46,7 +45,7 @@
   "peerDependencies": {
     "antd": "^4.2.0",
     "react": "^16.0.0",
-    "@ferlab/style": "^0.0.9"
+    "@ferlab/style": "^0.1.0"
```

Comment:
dont we need to change to 1.0.0 ?

Reply:
technically if its ready for production it should be 1.x.x an up